src/deployment/deploy.py

```diff
@@ -289,31 +293,25 @@ def create_password(self, object_id: UUID) -> Tuple[str, str]:
             "cli_password", object_id, self.get_subscription_id()
         )
 
-    def get_instance_url(self) -> str:
+    def get_instance_urls(self) -> List[str]:
         # The url to access the instance
         # This also represents the legacy identifier_uris of the application
         # registration
         if self.multi_tenant_domain:
-            return "https://%s/%s" % (
-                self.multi_tenant_domain,
-                self.application_name,
-            )
+            return ["https://%s/%s" % (self.multi_tenant_domain, name) for name in [self.application_name, self.application_name + DOTNET_APPLICATION_SUFFIX]]
```

Comment:
The "-net" functions are supposed to be temporary until we move everything to dotnet. 
with this changes the "-net" identifierUris and related info will become useless after the transfer.
Can you make this an option specified at deployment.

Reply:
@chkeita I think we should be setting this as long as the `deploy.py` script is deploying the `-net` application separately. Would it be sufficient to add a note next to the application deployment to say to remove this setting once we remove the Python app?

Reply:
i am thinking about upgrade scenarios for existing deployments. 
If we keep it in the code as-is, we need to add a the code to remove it afterwards when we do the migration. 
We already have a setting in the deployment script that specify which -net function we want to activate. We could make that it dependent on that.

Reply:
The other options is to make sure we update the identifierUris (and related settings) at each deployment. Currently we only set it on the first time we create the registration.

Reply:
@chkeita at the moment we do perform an update to add new identifier URIs ([see below](https://github.com/microsoft/onefuzz/pull/2090/files#diff-0489104f2cf3e8ff8968514b328b6cb496d9519ee74fa9f24b612ec060f2b4cfR452-R464)) but we never remove any. We could remove them in future?